DDDEastAnglia/Areas/Admin/Controllers/SponsorController.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Drawing;
+using System.Drawing.Imaging;
+using System.IO;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using DDDEastAnglia.DataAccess;
+using DDDEastAnglia.DataAccess.SimpleData.Models;
+using DDDEastAnglia.Helpers;
+using SponsorModel = DDDEastAnglia.Areas.Admin.Models.SponsorModel;
+
+namespace DDDEastAnglia.Areas.Admin.Controllers
+{
+    public class SponsorController : Controller
+    {
+        private readonly ISponsorRepository sponsorRepository;
+        private readonly DefaultSponsorSorter _defaultSponsorSorter;
+
+        public SponsorController(ISponsorRepository sponsorRepository, DefaultSponsorSorter _defaultSponsorSorter)
```

Comment:
[It looks like we used to pass in an `ISponsorSorter`](https://github.com/adrianbanks/dddeastanglia/compare/add-sponsors-to-db...CraftyFella:pull-sponsors-for-sidebar#diff-67f297aaf446858ad772daa1dafa8025L20): why are we now passing the concrete type?